jdaviz/configs/imviz/tests/test_catalogs.py

```diff
@@ -99,3 +101,47 @@ def test_plugin_image_with_result(self, imviz_helper):
         catalogs_plugin.vue_do_clear()
 
         assert not catalogs_plugin.results_available
+
+        # test loading from file
+        table = imviz_helper.app._catalog_source_table
+        skycoord_table = SkyCoord(table['ra'],
+                                  table['dec'],
+                                  unit='deg')
```

Comment:
I don't know if anyone (other than me) would benefit from an extra comment here, but I'll suggest it anyway. We could say that the `table` you're manipulating here is the default result from an SDSS catalog query, via astroquery. I was confused because you specify the `unit='deg'`, but in the title of the PR you say "from ecsv", which supports units. So I spent a while in the rabbit hole of checking where `table` stems from. In the end, it looks good. The ecsv source does not have units (for now).

Reply:
Ah yes, for context: this test was just an attempt to mimic the same format as the JWST ecsv files (at least as far as the one column that is actually parsed) without requiring a new remote data download in the tests.  So I used the SDSS query and write to an ecsv file using the same column structure and use that for the tests to ensure the same results are found.